src/home_work2/task1.js

```diff
@@ -0,0 +1,76 @@
+import * as Joi from '@hapi/joi';
+const express = require('express');
+const collection = require('./db');
+const bodyQuerySchema = require('./validations');
+const app = express();
+
+const bodyParser = require('body-parser');
```

Comment:
Consider finishing all the imports before your app main code.
And also group and separate your imports (new line).